src/lib/components/Cytoscape.react.js

```diff
@@ -37,8 +40,8 @@ class Cytoscape extends Component {
 
         // Trim down the element objects to only the data contained
         const edgesData = ele.connectedEdges().map((ele) => {
-                return ele.data();
-            }),
+            return ele.data();
+        }),
```

Comment:
nonblocking, but did the linter change the indentation here? Makes the following items look weird 🙃 Could probably clean that up by making each a separate statement. Also they're all mapping the same function 🌴